src/Microsoft.IdentityModel.Protocols.OpenIdConnect/LogMessages.cs

```diff
@@ -40,6 +40,7 @@ internal static class LogMessages
 
         // properties, configuration 
         internal const string IDX10105 = "IDX10105: NonceLifetime must be greater than zero. value: '{0}'";
+        internal const string IDX10106 = "IDX10106: Error in deserializing json: '{0}' into Newtonsoft.JObject.";
```

Comment:
We shouldn't specify 'Newtonsoft' here. Just inform user that the 'json' was not well formed.